test/read-file.test.js

```diff
@@ -101,77 +155,155 @@ describe('+ readFile()', function () {
         bothDone = true
       })
     })
+
+    it('should reject the promise', function (done) {
+      jf.readFile(file)
```

Comment:
Should have `{ throws: true }` set here.

Reply:
done